Incompatibility with `image 0.25.0`

Upgrading to `image 0.25.0` results in a compatibility issue between the `image::Pixel` trait implementation and the `render::Pixel` trait:

```rust
// issue with image::Luma<u8>'s implementation of Pixel
let img_buf = qr_code.render::<image::Luma<u8>>().build();
```

The most notable `image` change is the switch from the use of `#[repr(C)]` to `#[repr(transparent)]` in the Pixel trait implementation: 

```rust
#[derive(PartialEq, Eq, Clone, Debug, Copy, Hash)]
#[repr(transparent)]  // <- previously #[repr(C)]
#[allow(missing_docs)]
pub struct $ident<T> (pub [T; $channels]);  
```

dependencies
```
qrcode = "0.13.0"
image = "0.25.0"
```

use code
```
    // Encode some data into bits.
    let code = QrCode::new(b"01234567").unwrap();

    // Render the bits into an image.
    let image = code.render::<Luma<u8>>().build();

    // Save the image.
    image.save("/tmp/qrcode.png").unwrap();
```

error info
```
error[E0277]: the trait bound `Luma<u8>: Element` is not satisfied
   --> src/main.rs:44:31
    |
44  |     let image = code.render::<Luma<u8>>().build();
    |                      ------   ^^^^^^^^ the trait `Element` is not implemented for `Luma<u8>`
    |                      |
    |                      required by a bound introduced by this call
    |
    = help: the following other types implement trait `Element`:
              char
              &'a str
    = note: required for `Luma<u8>` to implement `qrcode::render::Pixel`
note: required by a bound in `QrCode::render`
   --> /Users/aya/.cargo/registry/src/rsproxy.cn-8f6827c7555bfaf8/qrcode-0.13.0/src/lib.rs:249:22
    |
249 |     pub fn render<P: Pixel>(&self) -> Renderer<P> {
    |                      ^^^^^ required by this bound in `QrCode::render`

```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        rustc: [1.61.0, stable] # MSVR and current stable rustc
+        rustc: [1.67.1, stable] # MSVR and current stable rustc
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,10 +17,10 @@ exclude = [
 maintenance = { status = "passively-maintained" }
 
 [dependencies]
-image = { version = "0.24", default-features = false, optional = true }
+image = { version = "0.25", default-features = false, optional = true }
 
 [dev-dependencies]
-image = "0.24"
+image = "0.25"
 
 [features]
 default = ["image", "svg"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -4,6 +4,7 @@ description = "QR code encoder in Rust"
 license = "MIT OR Apache-2.0"
 version = "0.13.0"
 edition = "2021"
+rust-version = "1.67.1"
 authors = ["kennytm <kennytm@gmail.com>"]
 keywords = ["qrcode"]
 repository = "https://github.com/kennytm/qrcode-rust"

